pkgs/development/python-modules/async/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchPypi, nose
+, zlib }:
+
+# The only user of this is "appdaemon" and we should drop it when appdaemon moves away from it:
+# https://github.com/home-assistant/appdaemon/issues/246
```

Comment:
I'm wondering why ofborg labeled 11-100 rebuilds, then. However, I cannot find any other users of async as well